### PR TITLE
Fixes #2452

### LINKED ISSUE #2452: Uncharger incorrectly removing charge from boron anions
**Description:**
```
In [12]: Chem.MolToSmiles(uncharger.uncharge(Chem.MolFromSmiles('C[B-](C)(C)C')))                                                                
[15:16:26] Running Uncharger
[15:16:26] Removed negative charge.
Out[12]: 'C[BH](C)(C)C'
```

### PR DESCRIPTION
**Pull request by greglandrum**
Adds a special case for neutralizing atoms that are early in the PT

### CODE REVIEW

File: Code/GraphMol/Atom.cpp [hunk 1]
```diff
@@ -22,12 +22,12 @@
 #include <RDGeneral/Dict.h>
 
 namespace RDKit {
-namespace {
+
 // Determine whether or not a molecule is to the left of Carbon
 bool isEarlyAtom(int atomicNum) {
```
Comment (bp-kelley) [inline Code/GraphMol/Atom.cpp]: Should this be a func of the periodic table?

**Comment by greglandrum** [comment]
yes, yes it should...
I will do that

**Comment by greglandrum** [comment]
Actually... that would then make it so that it wouldn't really be right to do this in the patch release (since it would add to the public API of a class).
How about I do a second "enhancement" PR that moves the function to the `PeriodicTable` class after this one is merged. That way we can get this bug fix out and have things cleaned up for the next release?

**Comment by greglandrum** [comment]
It will affect anything that's "to the left" of C, i.e. anything with less than 4 valence electrons.
I don't think it's particularly common that we find structures with things like a `[Be-]` in them, but they would also be neutralized,

**Pull request merged**